packages/blocks/src/lib/navigation-map/components/waypoints.svelte

```diff
@@ -7,5 +7,6 @@ import { waypoints } from '../stores';
   <MapLibreMarker
     scale={0.7}
     lngLat={waypoint}
+    icon={undefined}
```

Comment:
Instead of `undefined` here we should add a default to the export in `marker.svelte`:

```ts
export let icon: HTMLElement | undefined = undefined;
```